src/systems/ec/implementations/vanilla/packages/ec-component-form-radio/ec-component-form-radio.scss

```diff
@@ -15,25 +15,19 @@
   $box-border-color: $ecl-color-grey-50,
   $box-border-color-disabled: $ecl-color-grey-50,
   $box-border-color-checked: $ecl-color-blue-100,
+  $box-border-color-hover: $ecl-color-blue-100,
   $box-background: $ecl-color-white-100,
   $box-background-disabled: $ecl-color-grey-5,
   $outline-width: 3px,
   $outline-color: $ecl-color-yellow-100,
   $invalid-border-color: $ecl-color-error,
-  $invalid-border-width: 2px
+  $invalid-border-width: 2px // DEPRECATED
```

Comment:
if it's deprecated, why don't you remove it?

Reply:
it has to be kept as it is used in deprecated css rules, which are used on ECL 2.9.0